module/Finna/src/Finna/Form/Form.php

```diff
@@ -101,6 +101,13 @@ class Form extends \VuFind\Form\Form
      */
     protected $record;
 
+    /**
+     * Form settings (from YAML without parsing)
+     *
+     * @var array
+     */
+    protected $formSettings;
```

Comment:
Koska tätä ei aseteta constructorissa, olisi hyvä tehdä jompikumpi:
- oletus nulliksi ja heitetään poikkeus, jos yritetään tehdä jotain, missä tietoja tarvitaan mutta on edelleen null
- laittaa oletus vaikka `protected $formSettings = [];`. Ja sitten käsitellä mahdollinen puuttuva elementti oikein,  esim. `foreach ($this->formSettings['fields'] ?? [] as $el) {`

Reply:
Korjattu jälkimmäisen vaihtoehdon mukaiseksi.